front-end/src/RouterConfig.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import App from "./App";
+import User from "./components/Content/User/User";
+
+export default function RouterConfig() {
+  return (
+    <>
+      <BrowserRouter>

```

Comment:
No need to use 
```JS
<>
</>
```

Please refer doc: https://ja.reactjs.org/docs/fragments.html